bootstrap.sh

```diff
@@ -1 +1,22 @@
 #!/bin/bash
+
+kubectl create namespace mysql
```

Comment:
Why you don't deploy namespace from [namespace.yml](https://github.com/gaupt/devops_todolist_kubernetes_task_9_working_with_statefuset/blob/develop/.infrastructure/namespace.yml) file